factories/expense.py

```diff
@@ -0,0 +1,17 @@
+import random
+import factory
+
+from house.models import House
+from expenses.models import Expenses
+
+
+class ExpenseFactory(factory.Factory):
+    class Meta:
+        model = Expenses
+
+    description = ""
+    house_name = factory.Iterator(House.objects.all())
+    category = factory.LazyAttribute(lambda x: random.choice(
+        [tup[1] for tup in Expenses.Category.choices]))
+    amount = factory.LazyAttribute(lambda x: random.randint(5000, 25000))
+    date = factory.Faker('date_object')
```

Comment:
I was getting an error:
RuntimeWarning: DateTimeField Expenses.date received a naive datetime (197
7-03-22 00:00:00) while time zone support is active.
  warnings.warn(

after some googling I found that the solution is to write the date field like this:
```suggestion
    date = factory.Faker('date_time', tzinfo=timezone.get_current_timezone())

```

Reply:
I knew there was a way to do that thanks man